src/components/Loader/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+const getLinkClass = ({ isActive }: { isActive: boolean }) => (
+  classNames('navbar-item', { 'has-background-grey-lighter': isActive })
+);
+
+export const NavBar = () => {
+  return (
```

Comment:
return jsx without word `return` (if linter allows)
check in all other components